models/DataObject/Localizedfield/Dao.php

```diff
@@ -139,6 +139,26 @@ public function save($params = [])
          * which is a great performance gain if you have a lot of languages
          */
         DataObject\Concrete\Dao\InheritanceHelper::setUseRuntimeCache(true);
+
+        $ignoreLocalizedQueryFallback = \Pimcore::getContainer()->getParameter("pimcore.objects.ignore_localized_query_fallback");
```

Comment:
```suggestion
        $ignoreLocalizedQueryFallback = \Pimcore\Config::getSystemConfiguration('objects')['ignore_localized_query_fallback'];
```
I think we don't need a container parameter here, we can straight use the config.